packages/amplify-e2e-core/src/configure/index.ts

```diff
@@ -27,6 +27,7 @@ export const amplifyRegions = [
   'eu-central-1',
   'ap-northeast-1',
   'ap-northeast-2',
+  'ap-northeast-3',
```

Comment:
I think this change should be excluded until the CLI has released the region option in `amplify configure`. Otherwise it could cause the API e2e test failure. (I am testing with latest version `12.2.3` and do not find the new region).

Reply:
Removed in the last revision.